image_topic: "/d400/color/image_raw"
cam_info_topic: "/d400/aligned_depth_to_color/camera_info"
vocabulary_path: "/home/master/talos_walk_ws/src/dbow_ros/vocabularies/orb_mur.fbow"
max_kf_rate: 20
max_kfs: 100